libraries/AP_RangeFinder/AP_RangeFinder_BLPing.cpp

```diff
@@ -61,10 +61,9 @@ bool AP_RangeFinder_BLPing::get_reading(float &reading_m)
     } averageStruct;
 
     // read any available lines from the lidar
-    int16_t nbytes = uart->available();
-    while (nbytes-- > 0) {
-        const int16_t b = uart->read();
-        if (b < 0) {
+    for (auto i=0; i<8192; i++) {
```

Comment:
not keen on auto in this particular use case, '0' is a small signed integer, could end up as int8_t ?

Reply:
Read the docs and 0 is an `int`, so we're good.